.github/workflows/image-build-and-publish.yml

```diff
@@ -56,3 +56,65 @@ jobs:
           TAG=$(echo "${{ steps.version-string.outputs.tag }}" | sed 's/+/_/g')
           # Sign the ko image
           cosign sign -y $BASE_REPO:$TAG
+
+  operator-image-build-and-publish:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      packages: write
+      id-token: write
+
+    env:
+      BASE_REPO: "ghcr.io/stackloklabs/toolhive/operator"
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: go.mod
+
+      - name: Install Task
+        uses: arduino/setup-task@v2
+        with:
+          version: 3.x
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Generate CRDs
+        run: task operator-manifests
+
+      - name: Compute version number
```

Comment:
Do we want to go with a semver versioning system now we have tagged releases? Or at least have a `-dev` version for main builds?

Reply:
Added some options now, wdyt?